tasks/taxon_id/contamination/task_krona.wdl

```diff
@@ -4,32 +4,32 @@ task krona {
   input {
     File kraken2_report
     String samplename
-    String docker = "us-docker.pkg.dev/general-theiagen/biocontainers/krona:2.7.1--pl526_5"
+    String docker = "us-docker.pkg.dev/general-theiagen/staphb/krona:2.8.1"
     Int memory = 8
-    Int cpu = 4
+    Int cpu = 2
     Int disk_size = 100
   }
   command <<<
     # Get VERSION
-    ktImportTaxonomy 2>&1 | sed -n '/KronaTools /p' | sed 's/^.*KronaTools //; s/ - ktImportTaxonomy.*//' | tee VERSION
+    ktImportText | grep "KronaTools" | cut -d' ' -f3 | tee VERSION
 
-    # Get taxonomy file 
-    ktUpdateTaxonomy.sh taxonomy
```

Comment:
@sage-wright any reason why this step is no longer needed? This was used to update the NCBI's taxonomy file.

Reply:
Now that I looked a little better, krImportText doesn't require a taxonomy file (source: https://github.com/marbl/Krona/wiki/Installing) so good call!